tooling/templatize/pkg/pipeline/run.go

```diff
@@ -100,7 +77,7 @@ func RunPipeline(pipeline *Pipeline, ctx context.Context, options *PipelineRunOp
 	if err != nil {
 		return nil, err
 	}
-	dir := filepath.Dir(pipeline.pipelineFilePath)
+	dir := filepath.Dir(options.PipelineFilePath)
```

Comment:
Maybe rename this to the execution directory & do the transform during option parsing. This code no longer knows anything about pipeline files.

Reply:
will consider it for additional refactoring after this PR. Think this would be scope creep for this refactoring.

Reply:
I'll let it slide, but I think you're wrong - you are _adding_ this to options in this PR (!!!)